ui/stories/masked-input.md

```diff
@@ -10,16 +10,12 @@
 | [placeholder] | <code>String</code> | <code>value</code> | The placeholder to display before the user has entered any input. |
 | [allowCopy] | <code>bool</code> | <code></code> | Whether or not the input should render with a copy button. |
 | [displayOnly] | <code>bool</code> | <code>false</code> | Whether or not to display the value as a display only `pre` element or as an input. |
+| [onChange] | <code>function</code> \| <code>action</code> | <code>false</code> | A function to call when the value of the input changes. |
 
 **Example**
   
 ```js
 <MaskedInput
- @value={{attr.options.defaultValue}}
```

Comment:
looks like the example code here got cut off! unfortunately this is one of the bugs with jsdoc2md -- it doesn't like it when components are on multiple lines. unfortunately you'll have to copy/paste the example from the component source file. womp. :/

Reply:
Oh no :( I'll fit that, that's rather unfortunate though